samutev.sh

```diff
@@ -28,232 +34,251 @@ if [ ${RET} -ne 0 ]; then
   exit 42
 fi
 
-function help {
-      echo "Usage:"
-      echo "    $0 -h               Display this help message."
-      echo "    $0 -n <VM>          new <VM> with masterless minion"
-      echo "    $0 -s <VM>          new <VM> with minion and salt master. FIRST vm=saltmaster. minimum 2 vm's"
-      echo "    $0 -d <VM>          delete <VM>"
-      echo "    $0 -l               list   VM's"
-      echo
-      echo "Examples:"
-      echo "    $0 -n  testvm                                launch new testvm              as masterless minion"
-      echo "    $0 -n 'testvm1 testvm2 testvm3'              launch multiple new testvm's   as masterless minions"
-      echo "    $0 -n 'testvm1:c2:m1:d3 testvm2:c4:m2'  launch multiple new testvm's   as masterless minions"
-      echo "                                            with special settings for cpu, memory and disk"
-      echo "                                            testvm1 with 2 cpu, 1GB memory and 3GB disk"
-      echo "                                            defaults are c2 m1 d3"
-      echo
-      echo "    $0 -s 'salt-master1 testvm1 testvm2 testvm3'"
-      echo "                                        launch a saltmaster with multiple new testvm's"
-      echo "                                        FIRST vm = saltmaster"
-      echo "                                        minimum = 2 vm's"
-      echo "    $0 -s 'salt-master1:c2:m2:d6 testvm1'"
-      echo
-      echo "    $0 -d  testvm                       delaunch/delete testvm"
-      echo "    $0 -d 'testvm1 testvm2 testvm3'     delaunch/delete multiple testvm's"
-      echo
+function help() {
```

Comment:
tabs can have differentspaces, e.g. 2 or 4.
The result will we be different formatted usage, for one user it looks ok, for the next it looks ugly.

Reply:
Hmm. I dodn't know that. Where is defined how many spaces one tab resembles? But does it matter in the end as long as it is consistent?
We can replace each tab with x spaces if you prefer that. I don't know how your space variant looked for you but in my case it did not work out. Columns were misaligned. That's why I started the change in the first place.

Reply:
vim: set tabstop=4
terminal: tabs - set tabs on a terminal
So, i guess, the best way to keep it properly formatted to use single spaces

Reply:
Well but honestly, that also didn't work out ;-) Looked terrible for me.

I would still say that it doesn't matter how many spaces resemble one tab. At least you can be sure everything is column-aligned. 

Thinking about it again I wonder - isn't that flexibility in terms of how many spaces a tab is the "magic" behind tabs? It is as many spaces as are required to reach the next column start (for the last tab only - which might not be a complete tab where "complete" might be defined differently).
-> maybe we take that offline? :-)

Reply:
As discussed I added a commit to settle this discussion.